webapp/src/components/task_creation_form/TaskForm.vue

```diff
@@ -116,6 +128,17 @@
           </div>
         </IconCard>
       </div>
+      <IconCard>
+        <template #title> How to join after ?</template>
+        <div>
+          <p>
+            After submitting the form, others will be able to join the task by going into the
+            <span @click="goToDiscollaborative()" class="underline font-bold cursor-pointer">DISCOllaborative page</span>, and uploading their private data. You can simulate multiple
+            people by opening a new tab in your browser you can have a detailed explanation About
+            how Disco works <span @click="goToInformation()" class="underline font-bold	cursor-pointer">here</span>.
```

Comment:
```suggestion
            After submitting the form, others will be able to join the task from the
            <span @click="goToDiscollaborative()" class="underline font-bold cursor-pointer">DISCOllaborative page</span>. You can find more explanations about
            Disco in the <span @click="goToInformation()" class="underline font-bold cursor-pointer">Information page</span>.
```
I would reword this way! Be careful not to use "upload" as the point of Disco is exactly the opposite: train models without uploading/sharing data. You can use "connect" or "plug in" for example